src/integration/kotlin/Kafka2Hbase.kt

```diff
@@ -68,4 +72,22 @@ class Kafka2Hbase : StringSpec({
         val counter = waitFor { hbase.getCount(topic) }
         counter shouldBe startingCounter
     }
+
+    "Malfomed messages are written to dlp topic" {
+        val topic = uniqueTopicName()
+
+        val body = "junk".toByteArray()
+        val timestamp = converter.getTimestampAsLong(getISO8601Timestamp())
+        producer.sendRecord(topic, "key3".toByteArray(), body, timestamp)
+
+        Thread.sleep(20000)
+        consumer.subscribe(mutableListOf(Config.Kafka.dlqTopic))
+        val records = consumer.poll(pollTimeout)
+
+        val malformedRecord = MalformedRecord(String(body), "Not a valid json")
+        val expected = Klaxon().toJsonString(malformedRecord)
+        val actual = String(records.elementAt(0).value())
+        expected shouldBe actual
+
+    }
```

Comment:
I'd like to see a test that valid messages are **not** being written to the DLQ. As in theory this test could pass but that's because _every_ message is sent to the DLQ. The new test doesn't need to check anything else except the DLQ is empty as the writing to HBase for valid messages is tested elsewhere already.

Reply:
As discussed i have modified test description  of existing tests to accommodate assertions 
on checking in dlq besides in hbase